src/core/src/type/element_type.cpp

```diff
@@ -71,6 +71,8 @@ inline TypeInfo get_type_info(ov::element::Type_t type) {
         return {64, false, false, false, "uint64_t", "u64"};
     case ov::element::Type_t::nf4:
         return {4, false, false, true, "nfloat4", "nf4"};
+    case ov::element::Type_t::string:
+        return {8 * sizeof(std::string), false, false, false, "string", "string"};
```

Comment:
@rkazants could you explain how size of string type was determined?

Reply:
We store only object in the allocated memory pointed by `_impl`. All internal stuff related to string content is managed by `std::string` Class.